megamek/src/megamek/client/ui/swing/GUIPreferences.java

```diff
@@ -115,6 +115,9 @@
     public static final String DISPLAY_POS_Y = "DisplayPosY";
     public static final String DISPLAY_SIZE_HEIGHT = "DisplaySizeHeight";
     public static final String DISPLAY_SIZE_WIDTH = "DisplaySizeWidth";
+    public static final String GAME_SUMMARY_BOARD_VIEW = "GameSummaryBoardView";
+    public static final String GAME_SUMMARY_MINI_MAP = "GameSummaryMiniMap";
+    public static final String ENTITY_OWNER_LABEL_COLOR = "EntityOwnerLabelColor";
```

Comment:
Is this property related? It appears unused.

Reply:
`ENTITY_OWNER_LABEL_COLOR` comes from the merge: I assume it was a property that was added at line 118, the same line I added for the two properties related to game summary.  As far as I can tell, all three of those properties are referenced in the code.  I don't see anything in this addition that is unused.

Reply:
Gotcha, weird that GH marked it as an addition!